packages/terra-form/src/components/ChoiceField.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jsx-a11y/label-has-for */
+
+import React, { PropTypes } from 'react';
+
+import Control from './Control';
+import Field from './Field';
+
+const propTypes = {
+  /**
+   * Choices to populate radio buttosn for
+   */
+  choices: PropTypes.array,
```

Comment:
I wish these conversations would have happened on the design as it would have been easier to follow... but the <X>Field components are convenience molecules to make it easy to create the proper inputs to capture the proper data. Nothing prevents anyone from generating forms in a style that leaks the implementation details of  HTML